src/test/java/nextstep/subway/acceptance/line/LineSectionAcceptanceTest.java

```diff
@@ -0,0 +1,241 @@
+package nextstep.subway.acceptance.line;
+
+import static nextstep.subway.acceptance.common.SubwayUtils.responseToId;
+import static nextstep.subway.acceptance.line.LineUtils.responseToStationNames;
+import static nextstep.subway.acceptance.line.LineUtils.지하철노선_생성;
+import static nextstep.subway.acceptance.line.LineUtils.지하철노선_생성_후_ID_반환;
+import static nextstep.subway.acceptance.line.LineUtils.지하철노선_조회;
+import static nextstep.subway.acceptance.line.SectionUtils.지하철구간_삭제;
+import static nextstep.subway.acceptance.line.SectionUtils.지하철구간_생성;
+import static nextstep.subway.acceptance.station.StationUtils.지하철역_생성;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.annotation.DirtiesContext;
+
+@DisplayName("구간 관리 기능")
+@SpringBootTest(webEnvironment = WebEnvironment.DEFINED_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+public class LineSectionAcceptanceTest {
+
+    private static final Map<String, Long> stationIds = new HashMap<>();
```

Comment:
상수 선언을 의도하신 거라면, 관례에 맞게 이름을 대문자로 작성해보면 어떨까요?
```suggestion
    private static final Map<String, Long> STATION_IDS = new HashMap<>();
```

Reply:
음...재사용하려던 목적으로 작성했는데 지금 보니 접근지정자도 private이고 Map을 사용하는 것 자체가 비효율적인 것 같습니다. 이 부분은 전체적으로 개선하도록 하겠습니다!